netbout-web/src/main/java/com/netbout/dynamo/DyAttachment.java

```diff
@@ -411,7 +413,16 @@ private static Bucket storage() {
                 Manifests.read("Netbout-S3Secret")
             );
         } else {
-            region = new MkRegion();
+            final Path temp;
```

Comment:
@dmzaytsev we could just make this a `File` and assign:

```java
temp = Files.createTempDirectory(DyAttachment.class.getSimpleName()).toFile()
```
and hence save one redundant instantiation for a `File` in `region = new MkRegion(temp.toFile());` since it would then just be `region = new MkRegion(temp);` and `temp.deleteOnExit()` before that, couldn't we ?